tests/phpunit/CRM/Financial/BAO/FinancialAccountTest.php

```diff
@@ -177,4 +177,42 @@ public function testGetFinancialAccountByFinancialTypeAndRelationshipCustomAdded
       CRM_Financial_BAO_FinancialAccount::getFinancialAccountForFinancialTypeByRelationship(2, 'Credit/Contra Revenue Account is'));
   }
 
+  /**
+   * Test getting deferred financial type.
+   */
+  public function testGetDeferredFinancialType() {
+    $params = array(
+      'name' => 'TestFinancialAccount_1',
+      'accounting_code' => 4800,
+      'contact_id' => 1,
+      'is_deductible' => 0,
+      'is_active' => 1,
+      'is_reserved' => 0,
+    );
+
+    $ids = array();
+    $financialAccount = CRM_Financial_BAO_FinancialAccount::add($params, $ids);
```

Comment:
This is a nice not a must - but generally using $this->callAPISuccess rather than the BAO create methods is preferred in the test suites because
1) it usually shortens code - things like 'is_active', is_reserved have defaults. You don't need to defne $ids
2) you get better error messages when something goes wrong
3) we are encouraging use of the api so not creating less preferred code practices is a good thing